FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/PersistentBottomSheetActivity.kt

```diff
@@ -321,7 +329,13 @@ class PersistentBottomSheetActivity : DemoActivity(), SheetItem.OnClickListener,
             }
 
 
-            R.id.bottom_sheet_item_flag -> showSnackbar(resources.getString(R.string.bottom_sheet_item_flag_toast))
+            R.id.bottom_sheet_item_flag -> {
+                showSnackbar(resources.getString(R.string.bottom_sheet_item_flag_toast))
+                item.title = item.title + "clicked"
```

Comment:
Tried to test this, Title was not changing, maybe views layout is fixed.
Also, it will keep appending clicked to the title. So, maybe we can use some static string here

Reply:
Test will work on the bottomsheet which is built using DefaultContentProvider.add(H/V)List().
custom content needs to be changed by the consumer itself.
So Click toggle bottomsheet button and  then try.

For appending clicked every time was done intentionally to check behaviour at any point of time